src/builder.js

```diff
@@ -30,18 +29,18 @@ class Builder {
     await this.dataModelEngine.addAdminAccount();
   }
 
-  async build(dependencies = {}, config = Config.default()) {
+  async build(dependencies = {}, config) {
```

Comment:
I would swap the two parameters or remove the default for the first. Currently to call this method you either `build()` which will give you a `{}, undefined` pair, or `build(dep, config)` which will give you a `dep, config` pair. Which is IMO counterintuitive.

Reply:
Yes that makes sense :+1: